daffodil-runtime1-unparser/src/main/scala/edu/illinois/ncsa/daffodil/processors/unparsers/PackedDecimalUnparsers.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package edu.illinois.ncsa.daffodil.processors.unparsers
+
+import edu.illinois.ncsa.daffodil.processors.parsers.HasKnownLengthInBits
+import edu.illinois.ncsa.daffodil.processors.ElementRuntimeData
+import edu.illinois.ncsa.daffodil.processors.ParseOrUnparseState
+import edu.illinois.ncsa.daffodil.util.DecimalUtils._
+import java.lang.{ Long => JLong }
+import edu.illinois.ncsa.daffodil.processors.parsers.HasRuntimeExplicitLength
+import edu.illinois.ncsa.daffodil.processors.Evaluatable
+import edu.illinois.ncsa.daffodil.schema.annotation.props.gen.LengthUnits
+import edu.illinois.ncsa.daffodil.schema.annotation.props.gen.BinaryNumberCheckPolicy
+
+abstract class PackedNumberBaseUnparser(
+  e: ElementRuntimeData,
+  binaryPackedSignCodes: String,
+  binaryNumberCheckPolicy: BinaryNumberCheckPolicy)
+  extends PackedBinaryBaseUnparser(e, binaryPackedSignCodes, binaryNumberCheckPolicy) {
+
+  def fromBigInteger(num: String, nBits: Int, signCodes: String, policy: BinaryNumberCheckPolicy): Array[Byte] = packedFromBigInteger(num, nBits, signCodes, policy)
```

Comment:
I've found it's usually good practice to include the object for these static functions, so ``DecimalUtils.packedFromBigInteger(...)``. Just makes it more clear to see what function is being called.